BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/SkontoViewController.swift

```diff
@@ -125,6 +131,7 @@ public class SkontoViewController: UIViewController {
         notAppliedGroupView.addSubview(notAppliedView)
         view.addSubview(proceedView)
 
+        proceedView.proceedButton.addTarget(self, action: #selector(proceedButtonTapped), for: .touchUpInside)
```

Comment:
[Suggestion] lets keep addTarget inside view and just inject action in view init (example will be in a comment below)